packages/react/react/src/components/Portal/Portal.tsx

```diff
@@ -0,0 +1,80 @@
+import { createContext, useCallback, useContext, useLayoutEffect, useMemo } from 'react';
+import { createPortal } from 'react-dom';
+import { useIsMounted } from '../../hooks/useIsMounted';
+
+interface PortalProps {
+  children: React.ReactNode;
+  className?: string;
+  containerRef?: React.RefObject<HTMLElement | null>;
+}
+
+const portalContext = createContext<{ parentPortal: HTMLElement | null }>({
+  parentPortal: null,
+});
+
+const PORTAL_DEFAULT_CLASS = 'portal';
+
+function RenderPortal({ children, className, containerRef }: PortalProps) {
+  const { parentPortal } = useContext(portalContext);
+
+  const getPortalNode = useCallback(
+    (mountNode: HTMLElement) => {
+      const portalNode = mountNode.ownerDocument.createElement('div');
+      portalNode.classList.add(className || PORTAL_DEFAULT_CLASS);
```

Comment:
What do you think is better?
If you're going to set it as the default anyway, how about setting it from the beginning?

```suggestion
function RenderPortal({ children, className = PORTAL_DEFAULT_CLASS, containerRef }: PortalProps) {
  const { parentPortal } = useContext(portalContext);

  const getPortalNode = useCallback(
    (mountNode: HTMLElement) => {
      const portalNode = mountNode.ownerDocument.createElement('div');
      portalNode.classList.add(className);
```